src/styles/app.scss

```diff
@@ -52,6 +52,19 @@ code {
   border-radius: 4px;
   overflow: auto;
 }
+pre {
+  background: rgba(0, 0, 0, 0.3);
+  padding: .5rem;
+  line-height: 1.2;
+}
+pre>code {
+  line-height: 1.2;
+  padding: 0;
+  white-space: pre;
+  border: none !important;
+  background: transparent !important;
+  margin: 0;
+}
```

Comment:
I think a simpler change would get the same effect.  When I added the following rule to the page in Chrome the issue went away:
```css
pre > code {
  display: block;
  white-space: pre;
}
```